src/Http/Http.Results/src/ProblemHttpResult.cs

```diff
@@ -53,7 +53,7 @@ public Task ExecuteAsync(HttpContext httpContext)
             httpContext.Response.StatusCode = code;
         }
 
-        return HttpResultsHelper.WriteResultAsJsonAsync<object?>(
+        return HttpResultsHelper.WriteResultAsJsonAsync(
```

Comment:
And this change

Reply:
Since I am updating the HttpResultsHelper this change is not needed anymore.